scripts/prepare_sector_network.py

```diff
@@ -5283,23 +5283,28 @@ def add_industry(
         lifetime=costs.at["cement capture", "lifetime"],
     )
 
-    # TODO Link properly Industry to H2 topology
-    nodes_ind_z2 = spatial.h2_tyndp.nodes[spatial.h2_tyndp.nodes.str.contains("Z2")]
-    nodes_ind_z2 = nodes_ind_z2[~(nodes_ind_z2.isin(["IBFI H2 Z2", "IBIT H2 Z2"]))]
-    nodes_ind = n.buses.loc[nodes_ind_z2].country.values + "00"
-    nodes_ind[nodes_ind == "IT00"] = "ITCN"
-    nodes_ind[nodes_ind == "DK00"] = "DKE1"
-    nodes_ind[nodes_ind == "LU00"] = "LUG1"
-    nodes_ind[nodes_ind == "NO00"] = "NOS0"
-    nodes_ind[nodes_ind == "SE00"] = "SE01"
+    if options["h2_topology_tyndp"]["enable"]:
+        # TODO Link properly Industry to H2 topology
+        nodes_ind_z2 = spatial.h2_tyndp.nodes[spatial.h2_tyndp.nodes.str.contains("Z2")]
+        nodes_ind_z2 = nodes_ind_z2[~(nodes_ind_z2.isin(["IBFI H2 Z2", "IBIT H2 Z2"]))]
+        nodes_ind = n.buses.loc[nodes_ind_z2].country.values + "00"
+        nodes_ind[nodes_ind == "IT00"] = "ITCN"
+        nodes_ind[nodes_ind == "DK00"] = "DKE1"
+        nodes_ind[nodes_ind == "LU00"] = "LUG1"
+        nodes_ind[nodes_ind == "NO00"] = "NOS0"
+        nodes_ind[nodes_ind == "SE00"] = "SE01"
+    else:
+        nodes_ind = nodes
+        nodes_ind_z2 = nodes + " H2"
```

Comment:
We probably want to name this more general like
```suggestion
        nodes_ind_h2 = nodes + " H2"
```